base.scss

```diff
@@ -163,6 +163,7 @@
 @import "./src/stories/Blocks/reservation-page/reservation-page-skeleton";
 @import "./src/stories/Blocks/loan-page/loan-page-skeleton";
 @import "./src/stories/Blocks/registration-page/registration-page";
+@import "./src/stories/Blocks/registration-formular-page/registration-formular-page";
```

Comment:
Maybe this should just be `registration-form-page/registration-form-page`? I don't think "formular" is an english word. It would either have to be "form" or "formula", but the latter is also used in other contexts.

Reply:
👍